nodeJS/express-basics/Express-Lesson-2.md

```diff
@@ -58,6 +58,6 @@ One final note before diving back into the tutorial.  Express does not care abou
 </div>
 
 ### Additional Resources
-This section contains helpful links to other content. It isn't required, so consider it supplemental if you want to explore further:
+This section contains helpful links to other content. It isn't required, so consider it supplemental for if you need to dive deeper into something.
```

Comment:
I browsed back and checked other pervious lessons. The line they used is "This section contains helpful links to other content. It isn’t required, so consider it supplemental for if you need to dive deeper into something." 

If you could change it to that I think it'd be best!